README.md

```diff
@@ -48,9 +49,9 @@ for a project.
 
 ### Specifying dependencies
 
-A top level key `dependencies` points to a mapping, which contains all
+Top-level key `dependencies` point to a mapping, which contain all
```

Comment:
Maybe *A top-level key `dependencies` that points to a mapping  containing all [snip]*?